components/brave_wallet/browser/eth_tx_manager_unittest.cc

```diff
@@ -28,6 +28,7 @@
 #include "brave/components/brave_wallet/browser/keyring_service.h"
 #include "brave/components/brave_wallet/browser/pref_names.h"
 #include "brave/components/brave_wallet/browser/tx_service.h"
+#include "brave/components/brave_wallet/common/brave_wallet.mojom-forward.h"
```

Comment:
Do we need this when brave_wallet.mojom.h is already included?

Reply:
That's VSCode tried to be smart)